src/transfers/getKycUrl.ts

```diff
@@ -55,21 +55,26 @@ export function getKycUrl(params: KycUrlParams) {
   // break apart and re-produce the URL
   const { origin, pathname, search, hash } = new URL(params.response.url);
 
-  let callback;
+  let callback = "";
 
-  if (isPostMessage(params)) {
-    callback = "postMessage";
-  } else {
+  if ((params as PostMessageParams).callback_url === "postMessage") {
+    callback = `${search ? "&" : "?"}callback=postMessage`;
+  } else if (
+    (params as CallbackUrlParams).callback_url &&
```

Comment:
Should this coercion be extracted to a separate const? It's being used 3 times here. It's easy to miss all places if a change proves necessary.